### ISSUE TITLE
Facility Locator card display changes when changing Facility Type filter

### ISSUE THREAD

**Issue by ryguyk** [department-of-veterans-affairs/va.gov-cms#13651]
**This is likely low priority.**

## Describe the defect
On the Facility Locator, search-result cards are changing when the "Facility type" filter is changed _before_ the "Search" button is clicked:
- Hiding operating status and Covid-19 operating status alerts.
- Underline vs. no underline on card title.
From a technical perspective, it appears that the search-result cards are looking to the state of the "Facility type" dropdown to determine how to display things. Instead, the re-rendering should happen only on submitting the search.

## To Reproduce
1. Go to https://staging.va.gov/find-locations/
2. Search (click Search button):
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/6863534/639b5668-d900-40fa-ad99-f5a2adbc6dd0)
3. Note Covid-19 operating status in results:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/6863534/b78e6022-3eba-4d47-82aa-cf2539e027b9)
4. Change "Facility type" (do not click Search button):
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/6863534/fe2923de-25e8-4e88-b781-1dc3ceb1fd45)
5. Note Covid-19 operating status disappears:
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/6863534/2b479f9f-9dbb-4bd9-ae62-50e0aefa7d64)

## AC / Expected behavior
Since the "Search" button has not yet been clicked, the results should not be adjusted. It might make sense that result cards display operating status for "VA health" facilities and not "Community providers", but the search results do not change until the "Search" button is actually clicked, so the cards being displayed should not be changed until the new search is submitted.

## Screenshots
See above.

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`

**Comment by jilladams** [comment]
@eselkin FYI this has been discussed as a potentially good candidate for a first coding task for you once your onboarding paperwork is done and your environment is ready. We'll discuss more in sprint planning.
